modules/ballerina-jms/src/test/java/NetJMSTest.java

```diff
@@ -0,0 +1,96 @@
+///*
+//*   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//*
+//*  WSO2 Inc. licenses this file to you under the Apache License,
+//*  Version 2.0 (the "License"); you may not use this file except
+//*  in compliance with the License.
+//*  You may obtain a copy of the License at
+//*
+//*    http://www.apache.org/licenses/LICENSE-2.0
+//*
+//* Unless required by applicable law or agreed to in writing,
+//* software distributed under the License is distributed on an
+//* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//* KIND, either express or implied.  See the License for the
+//* specific language governing permissions and limitations
+//* under the License.
+//*/
+//
+//import org.ballerinalang.bre.Context;
+//import org.ballerinalang.model.BallerinaFile;
+//import org.ballerinalang.model.values.BMessage;
+//import org.ballerinalang.model.values.BString;
+//import org.ballerinalang.model.values.BValue;
+//import org.ballerinalang.nativeimpl.actions.jms.utils.Constants;
+//import org.ballerinalang.nativeimpl.util.BTestUtils;
+//import org.ballerinalang.nativeimpl.util.TestCallback;
+//import org.ballerinalang.util.codegen.ProgramFile;
+//import org.ballerinalang.util.program.BLangFunctions;
+//import org.testng.Assert;
+//import org.testng.annotations.BeforeClass;
+//import org.testng.annotations.Test;
+//import org.wso2.carbon.messaging.DefaultCarbonMessage;
+//
+//import javax.jms.Session;
+//
+///**
+// * Test cases for ballerina.net.jms native functions.
+// */
```

Comment:
Why is this test case commented out?

Reply:
This was a commented out test case in ballerina-native module, we only moved those to a new module. Test cases have to be updated according to the recent connector changes